src/components/dotCMS/layout/Layout.tsx

```diff
@@ -6,7 +6,7 @@ import { PageContext } from '@/contexts'
 import { Nav } from './Nav'
 import Footer from './Footer'
 
-export const Layout: React.FC = ({ children }) => {
```

Comment:
Why?

Reply:
Oh, my bad! After I updated the `TypeScript` version it starts throwing errors for that `type`. I'll fix it.